rs/node_rewards/canister/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 INSERT NEW RELEASES HERE
 
+# 2025-08-30: Proposal 138288
+
+http://dashboard.internetcomputer.org/proposal/138288
+
+## Added
+
+* Add get_node_providers_rewards endpoint, to be used by the Governance to compute performance-based rewards.
```

Comment:
Looks like no corresponding unreleased_changelog.md?  

Not an issue, just confirming that's on purpose.